components/cluster/libqb/Makefile

```diff
@@ -39,16 +40,31 @@ CONFIGURE_OPTIONS += CFLAGS="$(CFLAGS)"
 CONFIGURE_OPTIONS += --enable-debug
 CONFIGURE_OPTIONS += --localstatedir=/var
 CONFIGURE_OPTIONS += --with-socket-dir=/var/run/hacluster
+CONFIGURE_OPTIONS += --enable-slow-tests
 
 COMPONENT_PREP_ACTION=(cd $(@D); $(ENV) $(CONFIGURE_ENV) $(CONFIG_SHELL) autogen.sh)
 
 COMPONENT_PRE_CONFIGURE_ACTION=($(CLONEY) $(SOURCE_DIR) $(@D)) 
 
+# I can't figure out why this don't work if 'make test' is used:
```

Comment:
how does it fail?

Reply:
Likely, oi-userland environment can influence this. 
Quick fixes: "unexport SHELLOPTS" or "ENV = /usr/bin/env -i" can help with this.